ksql-parser/src/test/java/io/confluent/ksql/parser/SqlFormatterTest.java

```diff
@@ -390,5 +396,40 @@ public void shouldFormatExplainStatement() {
 
     assertThat(result, is("EXPLAIN \nSELECT *\nFROM ADDRESS ADDRESS"));
   }
+
+  @Test
+  public void shouldFormatInsertValuesStatement() {
+    final String statementString = "INSERT INTO ADDRESS (NUMBER, STREET, CITY) VALUES (2, 'high', 'palo alto');";
+    final Statement statement = KsqlParserTestUtil.buildSingleAst(statementString, metaStore)
+        .getStatement();
+
+    final String result = SqlFormatter.formatSql(statement);
+
+    assertThat(result, is("INSERT INTO ADDRESS (NUMBER, STREET, CITY) VALUES (2, 'high', 'palo alto')"));
+  }
+
+  @Test
+  public void shouldFormatInsertValuesNoSchema() {
+    final String statementString = "INSERT INTO ADDRESS VALUES (2);";
+    final Statement statement = KsqlParserTestUtil.buildSingleAst(statementString, metaStore)
+        .getStatement();
+
+    final String result = SqlFormatter.formatSql(statement);
+
+    assertThat(result, is("INSERT INTO ADDRESS VALUES (2)"));
+  }
+
+  @Test
+  public void shouldNotParseArbitraryExpressions() {
+    // Given:
+    final String statementString = "INSERT INTO ADDRESS VALUES (2 + 1);";
+
+    // Expect:
+    expectedException.expect(ParseFailedException.class);
+    expectedException.expectMessage("mismatched input");
```

Comment:
Yes it is a parser error - this is what @hjafarpour wanted, which is why this change added literals to the grammar directly. And yes, this would be a valuable feature to have sometime in the future.